.golangci.yaml

```diff
@@ -155,6 +155,7 @@ linters-settings:
     local-prefixes: package-operator.run
   gosec:
     excludes:
+    - G115 # TODO: Truncating integer conversions.
```

Comment:
What does this TODO mean?

Reply:
It is an important lint but it is not within the scope of this PR to write helper methods for strict integer conversions so I will leave the check disabled until we come around to that.